repos/fdbt-site/src/pages/csvZoneUpload.tsx

```diff
@@ -54,10 +129,18 @@ export const getServerSideProps = (ctx: NextPageContextWithSession): { props: Us
             csvTemplateDisplayName: 'Download fare zone CSV template - File Type CSV - File Size 673B',
             csvTemplateAttachmentUrl: FareZoneExampleCsv,
             csvTemplateSize: '673B',
-            errors,
+            errors: [...errors],
             detailSummary: "My fare zone won't upload",
             csrfToken: getCsrfToken(ctx),
             backHref: '',
+            showPriceOption: true,
```

Comment:
Why are we showing something to do with prices? This is a zone upload, not a fare triangle upload.